Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -228,154 +220,64 @@ std::string HiresTexture::GenBaseName(const u8* texture, size_t texture_size, co
                                       size_t tlut_size, u32 width, u32 height, TextureFormat format,
                                       bool has_mipmaps, bool dump)
 {
-  std::string name = "";
-  bool convert = false;
-  if (!dump && s_check_native_format)
-  {
-    // try to load the old format first
-    u64 tex_hash = GetHashHiresTexture(texture, (int)texture_size,
-                                       g_ActiveConfig.iSafeTextureCache_ColorSamples);
-    u64 tlut_hash = tlut_size ? GetHashHiresTexture(tlut, (int)tlut_size,
-                                                    g_ActiveConfig.iSafeTextureCache_ColorSamples) :
-                                0;
-    name = StringFromFormat("%s_%08x_%i", SConfig::GetInstance().GetGameID().c_str(),
-                            (u32)(tex_hash ^ tlut_hash), (u16)format);
-    if (s_textureMap.find(name) != s_textureMap.end())
-    {
-      if (g_ActiveConfig.bConvertHiresTextures)
-        convert = true;
-      else
-        return name;
-    }
-  }
+  if (!dump && !s_check_new_format)
```

Comment:
Can we drop the s_check_new_format variable as well? Or, alternatively, skip hashing if s_textureMap is empty, as AFAICT it will only contain entries when this var is set.